invenio_records_rest/facets.py

```diff
@@ -143,11 +143,21 @@ def default_facets_factory(search, index):
         selected_facets = make_comma_list_a_list(
```

Comment:
Can you also add the `facets` key-value pair (if it's passed) into `urlkwargs`, so that pagination links are properly generated?